scripts/gufi_stats

```diff
@@ -1105,6 +1105,17 @@ OTHERS = [
     ['gid-size',              gid_size],
 ]
 
+
+# a helper function to show the gufi_query being executed. Helpful for debugging and education.
+def my_pretty_print(cmd_tokens):
+  formatted_string = ""
+  for index, token in enumerate(cmd_tokens):
+      if token.startswith('-') or index == len(cmd_tokens)-1:
+          formatted_string += '\n\t' + token
+      else:
+          formatted_string += ' ' + token
+  print(f"Executing {formatted_string}")
```

Comment:
No f strings. I am keeping these python files usable with python2 just in case some systems still dont have python3

Reply:
Makes sense. Changed.